docs/src/api/class-browsertype.md

```diff
@@ -197,6 +197,11 @@ Specify environment variables that will be visible to the browser. Defaults to `
 
 Slows down Playwright operations by the specified amount of milliseconds. Useful so that you can see what is going on.
 
+### option: BrowserType.launch.showUserInput
```

Comment:
I am hesitant with a public option. We are growing a list of options nobody will ever know about. Let's think of "one to rule them all" approach?